proxy/outprocess/src/module_loaders/outprocess_loader.c

```diff
@@ -24,344 +31,616 @@ DEFINE_ENUM_STRINGS(OUTPROCESS_LOADER_ACTIVATION_TYPE, OUTPROCESS_LOADER_ACTIVAT
 
 typedef struct OUTPROCESS_MODULE_HANDLE_DATA_TAG
 {
-	const MODULE_API* api;
+    const MODULE_API* api;
+
 } OUTPROCESS_MODULE_HANDLE_DATA;
 
+static VECTOR_HANDLE uv_processes = NULL;
+static THREAD_HANDLE uv_thread = NULL;
+static tickcounter_ms_t uv_process_grace_period_ms = 0;
```

Comment:
No, the API's related to this data fall outside the ModuleLoader API, and therefore the API's and their respective data must be static in order to be called from the gateway.